jolt-core/src/zkvm/instruction_lookups/booleanity.rs

```diff
@@ -30,10 +34,10 @@ const DEGREE: usize = 3;
 struct BooleanityProverState<F: JoltField> {
```

Comment:
Maybe there's a better way to handle this? For exmaple, what if we have phase 1 prover state, and phase 2 prover state? I just don't like everything being an option.